DNN Platform/Library/Application/AssemblyStatusAttribute.cs

```diff
@@ -26,27 +26,27 @@ public enum ReleaseMode
         /// <summary>
         /// Not asssigned
         /// </summary>
-        None,
+        None = 0,
```

Comment:
I Added explicit numbers here to ensure this enum interops correctly with the new `DotNetNuke.Abstractions` enum. This will prevent any accidental breaking changes if the enums are ever re-ordered or changed. When we are converting the enums it is a simple cast as the int value matches, there is no need for a complex conversion method.